.binder/postBuild

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+set -e
+
+./dev-install.sh
```

Comment:
Since the Binder is user facing, how about using a released version of `ipywidgets` instead?

This would mean adding `ipywidgets=7.5` to `environment.yml`.

Reply:
You can create a Binder image for any given commit in the repo. I would expect each of those to have the appropriate version of `ipywidgets` and `jupyter-widgets/jupyterlab-manager` installed.

If all goes well (and I'm currently not sure that it will), I would like to do the same thing for the docs on RTD, which would mean that the `latest` docs are actually using the `master` version of the software.

And then, we could create Binder links on the RTD pages which link from a specific version of the docs to the exact same version running on Binder.

Reply:
OK, for this use case then it makes sense to do a dev install.